README.md

```diff
@@ -249,6 +249,25 @@ Cloudant({ account: me, password: password }, function(err, cloudant, pong) {
 });
 ~~~
 
+**Note:** For legacy compatibility with older versions of the library using the
+initialization callback will always add the `cookieauth` plugin to the list of
+configured plugins. If you do not want to use `cookieauth` then you should not
+use this initialization callback, instead initalize the client without a
+callback and then call the [ping function](https://console.bluemix.net/docs/services/Cloudant/api/account.html#ping)
```

Comment:
ping function is now here: https://cloud.ibm.com/docs/services/Cloudant/api?topic=cloudant-account#ping

Reply:
Nice spot, looks like we had a lot of these around still.
Fixed in 1b10733822c893642bbe5cee9d773b623b1742f5